ensembl/htdocs/info/genome/compara/super_trees.html

```diff
@@ -0,0 +1,58 @@
+<html>
+<head>
+  <title>Super trees</title>
+  <meta name="author" content="web" />
+  <meta name="navigation" content="Comparative Genomics" />
+</head>
+<body>
+
+<h1>Super trees</h1>
+
+<p>
+We call super tree the structure that links multiple gene trees together.
+Both our <a href="homology_method.html">protein trees</a> and <a href="ncRNA_methods.html">ncRNA trees</a> resources contain super trees.
+</p>
+
+<p>They arise from two situations</p>
+<ol>
+  <li>a dynamic process to cater for large families,</li>
+  <li>the handling of some Panther sub-families</li>
+</ol>
+
+<p>
+In both cases (see below), we infer some homologies across the super tree, which means that two orthologues or paralogues may be in <em>different</em> gene trees.
+</p>
+
+<h2>Breaking down large families</h2>
+
+<p>
+Large families that would be too complex to analyse are recursively broken down with <a href="http://www.ncbi.nlm.nih.gov/pubmed/12424131">QuickTree</a>.
+The current limits are 1,500 for <a href="homology_method.html">protein trees</a> and 400 for <a href="ncRNA_methods.html">ncRNA trees</a>.
+This happens dynamically based on the gene count of each family.
+</p>
+
+<p>
+The process generates a set of sub-families, each smaller than the required size, and a reconciled binary tree that links those.
```

Comment:
```suggestion
The process generates a set of sub-families, each smaller than the required size, and a reconciled binary tree that links them.
```
Although not sure if this changes the meaning...